.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,15 @@
+name: Unit Tests
```

Comment:
Why do you have this pipeline if you already have `.github/actions/unit-tests.yml` and the CI pipeline?

Reply:
thanks, file removed